spec/kinde_sdk_spec.rb

```
@@ -74,20 +74,20 @@
         )
         .to_return(
           status: 200,
-          body: { "access_token": "eyJ", "expires_in": 86399, "scope": "", "token_type": "bearer" }.to_json,
+          body: { "access_token": "eyJ", "id_token": "test", "refresh_token": "test","expires_in": 86399, "scope": "", "token_type": "bearer" }.to_json,
           headers: { "content-type" => "application/json;charset=UTF-8" }
         )
     end
 
     it "calls /token url with proper body and headers" do
-      expect(described_class.fetch_tokens(code).keys).to eq(%w[scope token_type access_token refresh_token expires_at])
+      expect(described_class.fetch_tokens(code).keys).to eq([:access_token :id_token :expires_at :refresh_token :scope :token_type])
```

RuboCop (1.73)

[fatal] unexpected token tCOLON
(Using Ruby 3.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)